src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -14,30 +14,605 @@
  * limitations under the License.
  */
 
-export const human38stats = {
+type RawSpeciesStats = {
+  [key: string]: {
+    [genomeId: string]: {
+      [key: string]: string;
+    };
+  };
+};
+
+export const sampleData: RawSpeciesStats = {
   assembly_stats: {
-    chromosome_count: 23,
-    genome_size: 3096649726,
-    total_sequence_length: 4537931177
+    homo_sapiens_GCA_000001405_28: {
+      'Scientific name': 'Homo sapiens',
+      Sex: 'NA',
+      'Breed/Cultivar/Strain': 'NA',
+      'Taxonomy id': '9606',
+      'Assembly name': 'GRCh38.p13',
+      'Assembly accession': 'GCA_000001405.28',
+      'Assembly date': '2019-02-28',
+      'Contig N50': '56413054',
+      'Total genome length': '3272116950',
+      'Total coding sequence length': '34317174',
+      'Total gap length': '161368351',
+      'Spanned gaps': '661',
+      Chromosomes: '25',
+      'Toplevel sequences': '640',
+      'Component sequences': '36734',
+      '% GC': '38.87'
```

Comment:
The structure presented here is very UI-specific, where the key in the json object is the same string as the label in the UI. I strongly feel that the proper long-term approach is to design a data structure that is likely to come from the backend, then figure out how to transform this data structure into something consumable by the frontend, and finally pass the proposed data structure to the backend for implementation. I am almost positive the the backend will not want to format the keys of the JSON this way.

A conversation with the backend team might help decide on the best shape of the json. @ens-st3, @jgtate what is your advise?